src/quo/components/list/item.cljs

```diff
@@ -5,12 +5,22 @@
             [quo.gesture-handler :as gh]
             [quo.design-system.spacing :as spacing]
```

Comment:
you could use this component here instead of the list item as it works with bottom-sheets
https://github.com/status-im/status-mobile/blob/develop/src/quo2/components/list_items/menu_item.cljs

Reply:
Thanks for the suggestion John
Will check how it looks with it

Reply:
@briansztamfater I have used `quo2.menu-item` is a bottom-sheet and it works well but I would like to ask you this since you implemented the component. Do you think we could use this component in all list-items of a bottom-sheet or do we have to implement a list-item component to use in other sheet that are not related to `reaction-drawer`